test/pkgs/PythonEnginePackage/extra/DSPythonNet3Extension.deps.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
same here

Reply:
I tried mocking the engine and package, but I guess I am stuck at mocking the assembly that the python loads, as we are fetching the python engine assembly location and matching it with package root directory. So it mocking seems complex..

Reply:
Lets create a task to improve the test, without using the actual package in order to not hold the PR?

Reply:
https://jira.autodesk.com/browse/DYN-7644